pkg/cmd/connector/connector_type/list/list.go

```diff
@@ -112,6 +113,10 @@ func runUpdateCommand(opts *options) error {
 		}
 	}
 
+	start := (opts.page - 1) * opts.limit
+	end := start + len(rows)
+	opts.f.Logger.Info(fmt.Sprintf("[page %v : %v - %v]", opts.page, start, end))
```

Comment:
Internationalization ("Page") can be added

Reply:
Done 👍